.github/workflows/dapr.yml

```diff
@@ -78,8 +78,13 @@ jobs:
     env:
       ARTIFACT_DIR: ./release
       DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+      HELMVER: v3.0.3
```

Comment:
Let's upgrade to the latest one (3.1.2) - https://github.com/helm/helm/releases/tag/v3.1.2